front/README.md

```diff
@@ -39,36 +39,16 @@ Launches end to end tests.
 
 It requires:
 
-- Backend containers to be up: `docker compose build core redis editoast postgres`
-- Running front
-- [`poetry`](https://python-poetry.org/docs/#installation) to be installed
-
-Locally, to use the same configuration as the CI, you must change the environment variable to
-`CI=true`.
-
-This command will execute all tests defined. The command can be changed in
-[`test_e2e.py`](../tests/tests/test_e2e.py) to execute only one test with or without debug mode.
-`test_e2e.py` is executing [playwright command](https://playwright.dev/docs/test-cli) in a python
-subprocess. Example of modification to apply to launch only `stdcm-page.spec.ts` in debug mode with
-chromium:
-
-```diff
-    result = subprocess.run(
-        [
-            "yarn",
-            "--cwd",
-            "front",
-            "playwright",
-            "test"
-+           "stdcm-page.spec.ts",
-            "--reporter=line",
-+           "--debug",
-+           "--project=chromium"
-        ],
-        cwd=Path(__file__).parents[2],
-        check=False,
-    )
-```
+- Install playwright dependencies `npx --prefix ./front/ playwright install --with-deps`
```

Comment:
I found these links to be helpful, can we include them somewhere?
https://classic.yarnpkg.com/lang/en/docs/install/
https://nodejs.org/en/download/package-manager